Automattic/wp-calypso
# Pricing Grid: Optimisation to remove Lodash

#### Proposed Changes

* This PR is one among several stacked PRs that implements the new pricing grid design. In https://github.com/Automattic/wp-calypso/pull/71372, we introduced a new signup flow /start/onboarding-2023-pricing-grid that is only available in dev and wpcalypso environments.
* The design for the pricing grid can be found in 2e991-pb. Please keep the Figma design open in a separate tab.
* The parent PR is https://github.com/Automattic/wp-calypso/pull/71532.
* This PR removes lodash.

#### Testing Instructions



* Same as https://github.com/Automattic/wp-calypso/pull/71532, verify that responsive screens work well.
